contracts/interfaces/sablier/LockupLinear.sol

```diff
@@ -25,4 +25,31 @@ library LockupLinear {
         address account;
         uint256 fee;
     }
+
+    struct Stream {
+        address sender;
+        uint40 startTime;
+        uint40 endTime;
+        uint40 cliffTime;
+        bool cancelable;
+        bool wasCanceled;
+        address asset;
+        bool transferable;
+        uint128 totalAmount;
+        address recipient;
+    }
```

Comment:
Let's make sure that this struct (and any others that we've re-created into our various Sablier Interface files) are fully separated from our _production contracts_. Like, we have our actual contracts, and a handful of Mock contracts to assist with testing. In order to minimize our contract bytecode for the contracts we're deploying to mainnets, let's keep those different structs fully separate and import the necessary files into the necessary contracts.